tools/src/publish-packages/tasks/addTemplateTarball.ts

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs';
+import path from 'path';
+
+import { selectPackagesToPublish } from './selectPackagesToPublish';
+import { TEMPLATES_DIR } from '../../Constants';
+import logger from '../../Logger';
+import { packToTarballAsync } from '../../Npm';
+import { Task } from '../../TasksRunner';
+import { Parcel, TaskArgs } from '../types';
+
+/**
+ * Add template tarball to Expo package.
+ */
+export const addTemplateTarball = new Task<TaskArgs>(
+  {
+    name: 'addTemplateTarball',
+    dependsOn: [selectPackagesToPublish],
+  },
+  async (parcels: Parcel[]) => {
+    const expoPackage = parcels.find((parcel) => parcel.pkg.packageName === 'expo');
+
+    if (!expoPackage) {
+      return;
+    }
+
+    logger.info('\nCopying template tarball to Expo package...');
```

Comment:
i think we also need to ensure that we align the versions in the template package.json with the versions that we are publishing. basically what we do when running `et upt`